example_plugin.py

```
@@ -1,13 +1,46 @@
-from plugins.base_plugin import BasePlugin
+from mmrelay.plugins.base_plugin import BasePlugin
+from mmrelay.matrix_utils import matrix_client, bot_command
+from mmrelay.meshtastic_utils import connect_meshtastic
 
 class Plugin(BasePlugin):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)